glean-core/ffi-sys/examples/ffi-usage.rs

```diff
@@ -0,0 +1,34 @@
+use std::ffi::CString;
+
+use cfg_if::cfg_if;
+use glean_ffi_sys::*;
+
+cfg_if! {
```

Comment:
Oh, so this is sugar so we can have `else` instead of a big `#[cfg(not(any(` like [here](https://searchfox.org/glean/rev/ed1b6d3bcf54a82e8f43bfeb4fd9f1af3d758382/glean-core/src/system.rs#70)?

Reply:
Yup, though this is just the example and I think I even broke this with recent changes.